django_celery_results/migrations/0010_auto_20210416_1502.py

```diff
@@ -0,0 +1,103 @@
+# Generated by Django 3.2 on 2021-04-16 15:02
```

Comment:
remove this migration mile as well and freshly generate with the new changes in models